vars.tf

```diff
@@ -408,4 +408,10 @@ variable "custom_policy_description" {
   type        = string
   description = "Allows to override the custom Lambda policy's description"
   default     = "The custom policy for the Lambda application module execution role"
+}
+
+variable "additional_layers" {
+  type        = list(string)
+  description = "A List of additional layers to be added to the function"
```

Comment:
Can we change this to `A list of layer ARN's (with or without aliases) to add to all functions within the Module. Provides the ability to add dependencies for additional functionality such as monitoring and observability`.